tests/nested/classic/snapshots-with-core-refresh-revert/task.yaml

```diff
@@ -0,0 +1,62 @@
+summary: test snapshots work when core snap is refreshed and reverted
```

Comment:
Silly question (pardon my ignorance) - whats the reason behind testing this in the nested VM? Could we do this also in the standard tests? It seems we install the snapd deb so couldn't we do the same on a regular spread run?

Reply:
Thanks for reviewing this, in fact for this test it is not really needed to build the snapd.deb, I just did it because in a next PR I am building snapd during the suite praparation and it will be copied to the nested vm by default. The idea of this test is to validate we can refresh/revert core from edge to stable and snapshots are working. This test is gonna be executed when a new core in edge is released, and not for every change. 
As we refresh/revert the core snap it does not make sense to run it as part of the main suite where we test the snapd.deb  

Any suggestion to change the way I am trying to test that is wellcome